fuel-chain-config/Cargo.toml

```
@@ -0,0 +1,27 @@
+[package]
+name = "fuel-chain-config"
+version = "0.1.0"
```

version must match the other crates to pass CI verification during publish
```suggestion
version = "0.10.1"
```